data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	The report refers to recent goings-on in human space as the Human Civil War, and follows the political and military situation, with commentary about human public opinions, theories of past Pug interference, and effects of the war on the Hai. Apparently, there was a massive influx of human refugees to Hai space during the war, which strained their social support systems. What is most surprising is the level of detail the book has on fleet movements and internal political discussions.`
+			`	Some of the details in the report were secrets the Hai shouldn't have known about know about. You recognize some of the information as private war planning discussions known only to yourself and a handful of other people. While there are obvious knowledge gaps, the level of detail they knew of the command hierarchy, fleet movements, special ops missions, intelligence services, and weapons designs, is quite disturbing.`
+			`	This points to a vast intelligence network; it seems Hai aren't as naive as their cute furry exterior implies. You look up at the functionary, and she gives you a friendly little smile that seems more sinister now than it did a minute ago.`
+			`	Of course, if their intelligence agencies are so capable, shouldn't they have been able to solve their pirate problems?`
+				goto reading
+				to display
+					not "Hai Reveal [A11] Sound the Alarm: pirates book"
+			`	It seems that, although the Hai have trouble tracking small, independent, anarchist, groups, they have no trouble with large, established, hierarchies. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
+				goto reading
+				to display
+					has "Hai Reveal [A11] Sound the Alarm: pirates book"
+
+			label pirates
+			action
+				log `The Hai intelligence agencies' approach to pirates was to befriend the large anarchist groups, as if they were the head of an "anarchist hierarchy. This suggests they lack understanding of raw chaos; in essence, their greatest weakness is asymmetric warfare.`
+				set "Hai Reveal [A11] Sound the Alarm: pirates book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	The report begins with a brief summary of Hai involvement with anarchist groups, which dates back from the founding of the first anarchist colony, Freedom, around the same time as human first contact. Apparently, the Hai have had generally positive relations with the largest organizations on anarchist planets. The Hai trade humanitarian supplies or other goods for assistance from the anarchist planets, even ones as distant as Thule and Greenrock.`
```

Comment:
Wait what?
This is new and totally unexpected to me.. How do they keep their cover? In human ships?
The Unfettered trade with anarchists but it generally goes the other way, and I wouldnt expect the Hai to approve although this would facilitate the job of those who want to smuggle to unfet space

Reply:
This is discussed elsewhere in Hai Reveal. Whatever method they use to spy on the human authorities, is likely how they contact the pirates. The simplest option is just sending encrypted messages through the hyperspace relays. However, they do have contact with human merchants who can be paid to smuggle messages. Also, we know they can hide satellites in a manner that prevents humans from finding them (the hyperspace relay mission), probably by making them look like asteroids or similar debris.

So, yes, the txt already refers to them doing this, and there are many ways they could accomplish it.

Reply:
Also, the Unfettered have a lot of experience hiding their doings from the Fettered. I would expect special ops missions and smuggling operations to human space would be no exception to this.

Reply:
Well the unfet are in their own territpry, this is said to be done by Hai outside of theirs

Reply:
Yes, and the Hai on both sides of Wah Ki have a hundred years of experience hiding their doings from humans.

Reply:
Yes, by acting with proxies?

Reply:
I'm not revealing how the Hai intelligence service works in these reports. They are low-level (hazel-2) reports, which only provide results of the intelligence, not sources or capabilities.

Reply:
This is now explained in the PR description, "You only have hazel-2 clearance, which does not let you learn how the Hai intelligence service works; you get the results, but not the sources or capabilities."